environment.yml

```diff
@@ -1,20 +1,20 @@
-name: image-analysis-23
+name: image-analysis-24
 channels:
   - conda-forge
 dependencies:
-  - python=3.10
+  - python=3.12
   - pip
   - jupyterlab
   - jupytext
   - notebook
   - matplotlib
-  - scikit-image=0.21.0
-  - jupyter-book=0.15.1
-  - pooch=1.7.0
-  - scikit-learn=1.2.2
-  - ipywidgets=8.0.7
-  - seaborn=0.12.2
-  - napari=0.4.18
+  - scikit-image=0.24.0
+  - jupyter-book=1.0.0
+  - pooch=1.8.2
+  - scikit-learn=1.5.0
+  - ipywidgets=8.1.3
+  - seaborn=0.13.2
+  - napari=0.4.19.post1
   - cookiecutter
   - cellpose
```

Comment:
```suggestion
  - cellpose
  - pyqt
```

Reply:
💯 
for conda install of napari, need to specify QT backend now, just like with pip ([all] includes pyqt)
I'd stick with pyqt, it's the more robust one.